index.html

```diff
@@ -11,18 +11,34 @@
         <link rel="stylesheet" href="./styles/style.css" />
         <link rel="stylesheet" href="./styles/reset.css" />
         <link rel="stylesheet" href="./styles/global.css" />
+        <span Class=”italics”>Kimber</span> 
 
-        <title>Cool Web</title>
+
+        <title>About Kamie</title>
     </head>
 
     <body>
         <header>
-            <h1>Cool Web Project</h1>
+            <h1>About Kamie Project</h1>
         </header>
-
-        <main></main>
-
-        <footer>&copy; 2021 Aspiring Software Developer</footer>
+        <header>
+            <h2>Kamie Short (She/Her)- Canby, Oregon</h2>
+            
+        </header>
+        <main>
+            <p>This is picture of my dog Kimber, she is a silver lab and is the weirdest dog that I know. Fun fact:
```

Comment:
I can immediately tell from her facial expression how much of a goofy dog she is! I love it! 😍